packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.js

```diff
@@ -37,11 +37,103 @@ describe("LabeledTextField", () => {
         expect(wrapper).toHaveState("focused", false);
     });
 
-    it("disabled prop disables the input", async () => {
+    it("value state changes when the user types", () => {
+        // Arrange
+        const wrapper = mount(<LabeledTextField label="Label" />);
+        const input = wrapper.find("input");
+
+        // Act
+        const newValue = "New Value";
+        input.simulate("change", {target: {value: newValue}});
+
+        // Assert
+        expect(wrapper).toHaveState("value", newValue);
```

Comment:
It might be good to have a test that checks that the value of the `input` is also `"value"`.

Reply:
Nevermind, I see there already is one.